cmake/modules/FindSpdlog.cmake

```diff
@@ -2,34 +2,41 @@
 # -------
 # Finds the Spdlog library
 #
-# This will define the following variables:
+# This will define the following target:
 #
-# SPDLOG_FOUND - system has Spdlog
-# SPDLOG_INCLUDE_DIRS - the Spdlog include directory
-# SPDLOG_LIBRARIES - the Spdlog libraries
-# SPDLOG_DEFINITIONS - the Spdlog compile definitions
-#
-# and the following imported targets:
-#
-#   Spdlog::Spdlog   - The Spdlog library
+#   spdlog::spdlog   - The Spdlog library
 
-if(ENABLE_INTERNAL_SPDLOG)
+if(NOT TARGET spdlog::spdlog)
   include(cmake/scripts/common/ModuleHelpers.cmake)
 
   # Check for dependencies - Must be done before SETUP_BUILD_VARS
-  get_libversion_data("fmt" "target")
-  find_package(Fmt ${LIB_FMT_VER} MODULE REQUIRED)
+  # Todo: We might need a way to do this after SETUP_BUILD_VARS...
+  if(ENABLE_INTERNAL_SPDLOG)
+    get_libversion_data("fmt" "target")
+    find_package(Fmt ${LIB_FMT_VER} MODULE REQUIRED)
+  endif()
 
-  # Check if we want to force a build due to a dependency rebuild
-  get_property(LIB_FORCE_REBUILD TARGET fmt::fmt PROPERTY LIB_BUILD)
+  if(TARGET fmt::fmt)
+    # Check if we want to force a build due to a dependency rebuild
+    get_property(LIB_FORCE_REBUILD TARGET fmt::fmt PROPERTY LIB_BUILD)
+  endif()
 
   set(MODULE_LC spdlog)
   SETUP_BUILD_VARS()
 
+  # Darwin systems we want to avoid system packages. We are entirely self sufficient
+  # Avoids homebrew populating rubbish we cant control
```

Comment:
I recommend the leading prepositions: "On Darwin systems..."

And I'll be trying this on my 2013 macbook. 

```cmake
  # On Darwin systems we want to avoid system packages. We are entirely self
  # sufficient. Avoids homebrew populating rubbish we cant control.
  if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
    set(_spdlog_find_option NO_SYSTEM_ENVIRONMENT_PATH)
  endif()
```